pcdsdevices/aerotech.py

```diff
@@ -0,0 +1,693 @@
+"""
+Aerotech devices.
+"""
+import logging
+
+import numpy as np
+from ophyd import Component as Cpt
+from ophyd.signal import EpicsSignal, EpicsSignalRO, Signal
+from ophyd.utils import LimitError
+
+from .epics_motor import EpicsMotorInterface, MotorDisabledError
+from .utils import stop_on_keyboardinterrupt
+
+logger = logging.getLogger(__name__)
+
+
+class AerotechMotorError(Exception):
+    """
+    Exception raised when an action requiring the motor not have an error is
+    requested.
+    """
+    pass
+
+
+class AerotechMotorFaulted(AerotechMotorError):
+    """
+    Exception raised when an action requiring the motor not be faulted is
+    requested.
+    """
+    pass
+
+
+class AerotechMotorStopped(AerotechMotorError):
+    """
+    Exception raised when an action requiring the motor to not be stopped is
+    requested.
+    """
+    pass
+
+
+class AerotechMotor(EpicsMotorInterface):
```

Comment:
Once this is pared down, seems like it should go in `epics_motor.py` so the correct class can be chosen based on prefix.